src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -472,6 +473,28 @@ void GeneralPage::initLayout(SettingsLayout &layout)
             return fuzzyToInt(args.value, 0);
         });
 
+    layout.addTitle("Streamer Mode");
+    layout.addDescription(
+        "Chatterino can automatically change behavior if it "
+        "detects that \"OBS Studio\" is running.\nSelect which "
+        "things you want to change while streaming");
+
+    layout.addDropdown<std::underlying_type<StreamerModeSetting>::type>(
+        "Enable Streamer Mode", {"No", "Yes", "Detect OBS (win)"},
```

Comment:
change (win) to (Windows only)

Reply:
The issue with that is in the settings it won't show the entire text in the dropdown.

Reply:
then the other option would be to just not show that option if you're not on windows

Reply:
Changed that to Windows only and made the dropdown menu a bit wider, hope that's okay.